invoices/templates/weekly_report_hours.html

```diff
@@ -0,0 +1,47 @@
+{% extends "base.html" %}
```

Comment:
These seem to be almost exact copies of per-invoice report pages. If that is the case, please split common part to a separate file and include it.

Reply:
The common part is now included in "snippets/report_hours.html"